### PR TITLE
v0.3

### PR DESCRIPTION
**Pull request by jvaubourg**
- The decryption web interface of cubes with encrypted fs has to be updated, because of the recent security issue, fixed in the image
- Testing kernels was not removed for cubes with encrypted fs (because of a find error with the olinux board detection)
- Having /etc/crypttab did not mean that the root is encrypted (this was not compatible with external encrypted hard drives)
- Then general clean up:
  - code restructuring
  - more stringent bash options (-Eu -o pipefail)
  - stdout/stderr of commands are not dropped if --verbose (-x) is enabled
  - replaces wrong _linux-image-4*_ apt regex by _'^linux-image-4.*'_

### CODE REVIEW

**Comment by jvaubourg** [comment]
Ping after 1 week @Psycojoker

File: scripts/src/env [hunk 1]
```diff
@@ -0,0 +1,23 @@
+set -Eu -o pipefail
+
+export DEBIAN_FRONTEND=noninteractive
+export DEBCONF_NONINTERACTIVE_SEEN=true
+export LC_ALL=C LANGUAGE=C LANG=C
+
+readonly _ISVERBOSE="$([ "${-/x}" != "${-}" ] && echo true || echo false)"
+readonly _LATESTVERSION="$(cat ./version)"
```
Comment (Psycojoker) [inline scripts/src/env:8]: The way the shell handle pwd is so horrible. (not your fault, just me ranting)

**Comment by Psycojoker** [comment]
Everything seems fine for me, thx for your work :)

(it really has a feeling of you trying to do a haskellish way of building abstractions using bash)

**Comment by Psycojoker** [comment]
Do you feel like we can update our `apps.json`?

**Comment by jvaubourg** [comment]
Thank you for your review! For me, we can update `apps.json` and announce the release with the security issue fix.

**Comment by jvaubourg** [comment]
Wait! Maybe, in `install`, we should not set the commit id of the decryption web interface. If we do that, DoctorCube has to be updated each time we change the we interface, even for minor changes. Maybe we should update _to the latest_ in `install`?

**Comment by jvaubourg** [comment]
Done

**Pull request merged**